Objects/TI4_MultiRoller_Plat.ttslua

```diff
@@ -762,8 +762,15 @@ end
 
 function setUnitCount(unit, count)
     assert(type(unit) == 'string' and type(count) == 'number')
-    self.UI.setValue(unit .. ".unitCount", count)
-    _unitTypeToCount[unit] = count
+
+    -- Restrict to allowable unit types
```

Comment:
Is something setting an invalid unit type?

Reply:
Not invalid, but not a base unit type either. To inject a new unit type (like "The Cavalry" from the Nomad promissory note), it gets added to the list of unit attributes. Because there are no "The Cavalry" units in the hex, this is being used to set the number of "The Cavalry" units to 0. And then this entry takes precedence over the unit attributes "unitCount" property (which is 1).

Reply:
Gotcha.  Maybe add that to the comment for the next person to scratch their head about that check. :D